CMakeLists.txt

```diff
@@ -103,6 +104,13 @@ include_directories(${TILEDB_INCLUDE_DIR})
 #librt
 find_library(LIBRT_LIBRARY rt)
 
+#pgsql driver and dbi libs
+find_package(libdbi)
+if(LIBDBI_FOUND)
```

Comment:
It is optional. If libdbi is not installed in an environment, the dependent files won't be compiled.
SQLVidMapper in this instance.